util/network-devp2p/src/discovery.rs

```diff
@@ -145,6 +146,7 @@ pub struct Discovery<'a> {
 	discovery_id: NodeId,
 	discovery_nodes: HashSet<NodeId>,
 	node_buckets: Vec<NodeBucket>,
+	other_observerd_nodes: LruCache<NodeId, NodeEndpoint>, // Sometimes we don't want to add nodes to the NodeTable, but still want to
```

Comment:
typo: `observed` 
`[..] want to` what?

Reply:
Shit, second line of a multiline comment is lost; will fix, thanks for spotting.